### PR TITLE
Moving Jenkins pipline to run on parallel for k8s testing & onprem 

### PR DESCRIPTION
**Pull request by oshercc**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -9,67 +9,85 @@ pipeline {
...
+                        clear_slave()
+
+                        sh '''export PATH=$PATH:/usr/local/go/bin; make build-image build-assisted-iso-generator-image'''
+                        withCredentials([usernamePassword(credentialsId: 'dockerio_cred', passwordVariable: 'PASS', usernameVariable: 'USER')]) {
+                            sh '''docker login docker.io -u $USER -p $PASS'''
+                        }
+                        sh '''export PATH=$PATH:/usr/local/go/bin; make jenkins-deploy-for-subsystem'''
+                        sh '''kubectl get pods -A'''
+                        sh '''export PATH=$PATH:/usr/local/go/bin;make subsystem-run'''
+                        sh 'make clear-deployment'
```
Comment (filanov) [inline Jenkinsfile:33]: You will not be able to get any log after clear-deployment.

Comment (oshercc) [reply Jenkinsfile]: True

**Review by filanov — APPROVED** [review]

**Comment by filanov** [comment]
@oshercc of some reason it run two jobs is that OK?

**Pull request closed without merging**